docs/BEMAN_STANDARD.md

```diff
@@ -262,6 +262,18 @@ or
 ![Library Status](https://raw.githubusercontent.com/bemanproject/beman/refs/heads/main/images/badges/beman_badge-beman_library_retired.svg)
 ```
 
+Use exactly one of the following entries for the standard target status badge:
```

Comment:
Ok -- idk that it should be a requirement instead of a recommendation -- but I wouldn't want to complicate things.